src/node_report.cc

```diff
@@ -1030,14 +1039,20 @@ static void PrintResourceUsage(std::ostream& out) {
 #if defined(__APPLE__) || defined(_AIX)
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_utime.tv_sec, stats.ru_utime.tv_usec);
     out << "\n  User mode CPU: " << buf << " secs";
+    cpu_abs = std::stod(buf);
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_stime.tv_sec, stats.ru_stime.tv_usec);
     out << "\n  Kernel mode CPU: " << buf << " secs";
+    cpu_abs += std::stod(buf);
 #else
     snprintf( buf, sizeof(buf), "%ld.%06ld", stats.ru_utime.tv_sec, stats.ru_utime.tv_usec);
     out << "\n  User mode CPU: " << buf << " secs";
+    cpu_abs = std::stod(buf);
     snprintf( buf, sizeof(buf), "%ld.%06ld", stats.ru_stime.tv_sec, stats.ru_stime.tv_usec);
     out << "\n  Kernel mode CPU: " << buf << " secs";
+    cpu_abs += std::stod(buf);
 #endif
+    cpu_percentage = (cpu_abs / diff_time) * 100;
+    out << "\n  Average Consumption : "<< cpu_percentage << "%";
```

Comment:
Division with double value rather integer - done.
Modified the header to "Average CPU Consumption" 
I would think the CPU consumption across all the CPUs make sense. Having already averaged out the consumption across the whole duration of the run, the fine-grained information split by the threads may not be required, or meaningful. So in short, the current code stands.  Please let me know what is your thought on this.